### DISCUSSION TITLE
Connected. Waiting for traffic

### DISCUSSION THREAD

**Discussion by jupitert25** [engageub/InternetIncome#71] [Q&A]
![image](https://github.com/engageub/InternetIncome/assets/151444646/e10d81af-e599-4c06-b87e-6bacfa085c30)
Hi. it's weird . I just start doing new account with new device and this happen . it not running on honeygain. I return and use my old device to test if i setting wrong or something but still same issue . Do you know how to fix this ?

**Comment by engageub** [comment]
Hello,
If it shows the device there, it is working as expected. It may be a proxy issue.
Sometimes it may take about an hour. It depends on the location of the proxy also.

Thank you

**Reply by jupitert25** [reply]
it's been 24h still not connected but Earnapp is running ok. I think something wrong with Honey Gain docker version

**Reply by engageub** [reply]
Hi,
Please enable logs and see if it looks good. Else please send email to honeygain or ask in their discord group.
This happened earlier based on the location of the proxy. After changing the proxy it started working.
Thank you

**Reply by jupitert25** [reply]
so i change --platform=linux/amd64 honeygain/honeygain to honeygain/honeygain:0.6.6 in your internetIncome.sh and it working 
it's seem like thier new version honeygain docker (0.8.0) is not working or bugging , alot of people in discord got same issue .

**Reply by engageub** [reply]
Thank you for the update. InternetIncome script always pulls the recent version before starting.
From the history, it can be seen that honeygain has updated to 0.8.0 3 days back.
Lets wait for them to resolve any issues in the next version. The InternetIncome script is working as expected. 
It will be problem for maintenance if version numbers are specified in the script and later if that version gets old and deleted, each of the users using the script have to update. 

https://hub.docker.com/r/honeygain/honeygain/tags 

Thank you

**Reply by af0rte** [reply]
Hello jupiter,

its possible to share with me the discord group?

Thanks.

**Reply by jupitert25** [reply]
check it on their website earnapp they have discord the end of the page

**Comment by engageub** [comment]
Hello,
I have seen these errors recently when the proxies were not residential IP.
Please make sure the proxies are Residential IPs.
You may also look into honeygain container logs for errors. 

Thank you
